lxd/config/map.go

```diff
@@ -259,3 +259,57 @@ func normalizeBool(value string) string {
 func IsUserConfig(key string) bool {
 	return strings.HasPrefix(key, "user.")
 }
+
+// IsProjectStorageConfig returns true if the key starts with the prefix "storage.project.".
+func IsProjectStorageConfig(key string) bool {
+	return strings.HasPrefix(key, "storage.project.")
+}
+
+// DaemonStorageType represents the type of storage volume configurable in the daemon config.
+type DaemonStorageType string
+
+// Define the possible types of daemon storage.
+const (
+	DaemonStorageTypeImages  DaemonStorageType = "images"
+	DaemonStorageTypeBackups DaemonStorageType = "backups"
+)
+
+// ParseDaemonStorageConfigKey parses a daemon storage config key and returns the project name
+// and the type of storage it refers to. It supports both project-specific keys
+// (e.g., "storage.project.<name>.images_volume") and daemon storage keys (e.g., "storage.images_volume").
+// If the key is not recognized, it returns an empty string and an empty DaemonStorageType.
+func ParseDaemonStorageConfigKey(config string) (projectName string, storageType DaemonStorageType) {
+	if strings.HasPrefix(config, "storage.project.") {
+		parts := strings.Split(config, ".")
+		if len(parts) != 4 {
+			return "", ""
+		}
+
+		switch parts[3] {
+		case "images_volume":
+			return parts[2], "images"
```

Comment:
Not a blocker, but please follow up by returning DaemonStorageTypeBackups here and DaemonStorageTypeImages in the line above.